crates/red_knot_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -705,6 +727,9 @@ impl<'db> UseDefMapBuilder<'db> {
         self.scope_start_visibility = self
             .visibility_constraints
             .add_or_constraint(self.scope_start_visibility, snapshot.scope_start_visibility);
+
+        // At least one of the two snapshots was reachable, so the merged result is too.
```

Comment:
If we've gotten past the top-of-function checks, we know that _both_ of the two states were reachable, right? So this comment understates what we know here.

Reply:
Yes good catch!  Stale comment from a previous iteration of this function.